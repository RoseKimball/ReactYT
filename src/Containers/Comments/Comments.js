import React, { Component } from 'react';
import CommentsHeader from './CommentsHeader/CommentsHeader';
import AddComment from './AddComment/AddComment';
import Comment from './Comment/Comment';
import { Divider } from 'semantic-ui-react';

export class Comments extends Component {
    render() {
        return (
            <div>
                <Divider />
                <CommentsHeader amountComments={this.props.amountComments}/>
                <AddComment />
                <Comment />
                <Comment />
                <Comment />
                <Comment />
                <Comment />
            </div>
        );
    };
};

export default Comments;