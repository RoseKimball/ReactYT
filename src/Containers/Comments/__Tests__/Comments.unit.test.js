import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import Comments from '../Comments';

describe('Comments', () => {
    test('renders Comments', () => {
        const wrapper = shallow (
            <Comments />
        )
        expect(wrapper).toMatchSnapshot();
    })
    test('renders Comments with amountComments prop', () => {
        const wrapper = shallow (
            <Comments amountComments={1}/>
        )
        expect(wrapper).toMatchSnapshot();
    }) 
}