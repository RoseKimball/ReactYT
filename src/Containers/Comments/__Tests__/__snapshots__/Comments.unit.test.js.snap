// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders Comments 1`] = `ShallowWrapper {}`;

exports[`Comments renders Comments with amountComments prop 1`] = `ShallowWrapper {}`;
